core/templates/components/forms/slider/audio-slider.component.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Wrapper over mat-slider for audio-bar.
+ */
+
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+@Component({
+  selector: 'oppia-audio-slider',
+  template: `<mat-slider
+    color="accent"
+    [value]="value"
+    style="width: 100%;"
+    [thumbLabel]="thumbLabel"
+    [max]="max"
+    (change)="setProgress($event)"
+    tick-interval="auto"
+    [step]="1"
+    aria-label="audio-slider">
+  </mat-slider>`,
+  styles: [`
+  .mat-accent /deep/ .mat-slider-track-fill,
+  .mat-accent /deep/ .mat-slider-thumb,
+  .mat-accent /deep/ .mat-slider-thumb-label {
+    background: #ff4081;
+  }`]
+})
+export class AudioSliderComponent {
+  @Input() value: number;
+  @Input() max: number;
+  @Input() thumbLabel = false;
+  @Output() valueChange = new EventEmitter<{ value: number }>();
+  constructor() { }
+
+  setProgress(event: {value: number}): void {
```

Comment:
is this number progress [0.0, 1.0] or duration of the audio? [For progress name in the function the 0.0, 1.0 makes more sense] (if it's duration then should we change the function name to setDuration?)

I think "progress" fits better with slider, wdyt?